packages/webcert/src/feature/certificate/NotificationBanners/SystemBanner.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { useSelector } from 'react-redux'
+import NotificationBannerBase from './NotificationBannerBase'
+import { getBanners } from '../../../store/utils/utilsSelectors'
+import { Banner } from '../../../store/utils/utilsReducer'
+
+const SystemBanner: React.FC = () => {
```

Comment:
Just a minor thing, but technically this component renders multiple banners and the name `SystemBanners` would convey that better